src/cli/project/name/mod.rs

```diff
@@ -0,0 +1,37 @@
+pub mod get;
+pub mod set;
+
+use crate::Project;
+use clap::Parser;
+use std::path::PathBuf;
+
+/// Commands to manage project name.
+#[derive(Parser, Debug)]
+pub struct Args {
+    /// The path to `pixi.toml` or `pyproject.toml`
+    #[clap(long, global = true)]
+    pub manifest_path: Option<PathBuf>,
```

Comment:
We just switched to use the `ProjectConfig` struct for this as much as possible so we only have on place to change the manifest_path. Check out the project/channel where I changed that a few days ago.

Reply:
Thanks! Taking a look.

Reply:
Thanks for your feedback @ruben-arts. I'm looking at that PR and was wondering, why does the declaration of `project` move out of the `execute` function in `mod.rs`? https://github.com/prefix-dev/pixi/pull/2569/files#diff-531dc39e5f51cb4dc5acf88fe6aa4e25f90a6d492724f3f8668d12f9f6cce9cbL107-R106

Can it stay and be passed to the `execute` functions on `get` and `set`?

```
let project = Project::load_or_else_discover(args.project_config.manifest_path.as_deref())?;

match args.command {
    Command::Get => get::execute(project).await?,
    Command::Set(args) => set::execute(project, args).await?,
}
```

Reply:
Good question! In this case we don't need to do something with the configuration, so we don't need to do it in the sub commands themselves. So it's fine as you build it already!